pkg/liqo-controller-manager/foreign-cluster-operator/setup.go

```diff
@@ -0,0 +1,32 @@
+// Copyright 2019-2021 The Liqo Authors
```

Comment:
AFAIU this file could be dropped, as already done in the liqo-controller-manager main